docs/concepts/workloads/pods/init-containers.md

```diff
@@ -129,6 +129,55 @@ spec:
     command: ['sh', '-c', 'echo The app is running! && sleep 3600']
 ```
 
+There is a slight change of syntax in Kubernetes 1.6. We move declaration of the init containers to `spec`:
```

Comment:
replace: "There is a new syntax in Kubernetes 1.6, although the old annotation syntax still works.  We have moved the declaration of init containers to `spec`:"